BoxContentSDK/BoxContentSDK/Requests/BOXFolderPaginatedItemsRequest.m

```diff
@@ -28,6 +28,18 @@ - (instancetype)initWithFolderID:(NSString *)folderID inRange:(NSRange)range
     return self;
 }
 
+- (instancetype) initWithFolderID:(NSString *)folderID metadataTemplateKey: (NSString *)metadataTemplateKey metadataScope: (BOXMetadataScope *)metadataScope inRange:(NSRange)range
+{
+    if (self = [super init]){
+        _folderID = folderID;
+        _limit = range.length;
```

Comment:
Nit: traditionally, pagination params (limit and offset) go after the item attribute params.